autosklearn/estimators.py

```diff
@@ -795,14 +797,18 @@ def fit(self, X, y,
         """
         # Before running anything else, first check that the
         # type of data is compatible with auto-sklearn. Legal target
-        # types are: continuous, binary, multiclass.
+        # types are: continuous, continuous-multioutput, 'multiclass'
```

Comment:
I think you need to update this.

Reply:
Same here, improved the message to show we need this.